backend/middleware/sanitize.js

```
@@ -0,0 +1,182 @@
+  'class',
+  'style',
+];
+
+// Create a DOMPurify instance with jsdom
+const window = new JSDOM('').window;
+const DOMPurify = createDOMPurify(window);
+
+// --- Task 9.2: Schema-driven, field-level sanitization utility ---
+
+const DEFAULT_URI_REGEXP =
+  /^(?:(?:(?:f|ht)tps?|mailto|tel|callto|sms):|[^a-z]|[a-z+.\-]+(?:[^a-z+.\-:]|$))/i;
```

ESLint

[error] Unnecessary escape character: \-.

(no-useless-escape)